docs/products/cassandra/howto/enable-cross-cluster-replication.md

```diff
@@ -1,76 +1,63 @@
+CLI, or programmatically with Aiven API.
 
-## Prerequisites
+## Prerequisites {#enable-ccr-in-aiven}
 
+-   This feature requires `[Aiven Enterprise](/docs/platform/howto/aiven-enterprise)`.
 -   This is a [limited availability feature](/docs/platform/concepts/beta_services). To try
     it out, contact the sales team at [sales@aiven.io](mailto:sales@aiven.io).
 -   Aiven account
 -   Depending on the method you choose to use for enabling CCR
     -   Access to [Aiven Console](https://console.aiven.io/)
     -   `cURL` CLI tool
     -   [Aiven CLI tool](https://github.com/aiven/aiven-client)
--   See [Limitations](/docs/products/cassandra/concepts/cross-cluster-replication#ccr-limitations).
+-   Check
+    [CCR limitations](/docs/products/cassandra/concepts/cross-cluster-replication#ccr-limitations).
 
 ## Enable CCR in the console
 
 Using [Aiven Console](https://console.aiven.io/), you can enable CCR for
 
 -   New Aiven for Apache Cassandra service by
-    [creating a totally new CCR service pair](/docs/products/cassandra/howto/enable-cross-cluster-replication#new-pair) or
+    [creating a totally new CCR service pair](/docs/products/cassandra/howto/enable-cross-cluster-replication#new-pair)
+    or
 -   Existing Aiven for Apache Cassandra service by
     [adding a CCR peer service in another region to an existing service](/docs/products/cassandra/howto/enable-cross-cluster-replication#new-peer).
 
 ### Create a new CCR service pair {#new-pair}
 
 1.  Log in to [Aiven Console](https://console.aiven.io/).
-2.  From the **Services** page, select **+Create service**.
-3.  In the **Create service** view
+1.  From the **Services** page, select **+Create service**.
+1.  In the **Create service** view
     1.  Select Apache Cassandra® and, next, a cloud provider and a cloud
         region for your service.
-    2.  To activate CCR, select the **Enable Cross Cluster Replication**
+    1.  To activate CCR, select the **Enable Cross Cluster Replication**
         toggle, which unfolds an additional section for setting up
         another Apache Cassandra service.
-    3.  Select a cloud provider and a cloud region for the second
+    1.  Select a cloud provider and a cloud region for the second
         service.
-    4.  Select one service plan for your two new services.
-    5.  Optionally, add an additional disk storage for your services.
-    6.  Enter names for your services.
+    1.  Select one service plan for your two new services.
+    1.  Optionally, add an additional disk storage for your services.
+    1.  Enter names for your services.
 
 :::note[Datacenters names]
 Notice that the service names you enter are automatically copied as the
 names for datacenters for the services. The datacenters names can be
 required when executing statements on your databases.
 :::
 
-4.  Verify your choices in the **Services summary** card on the right
+1.  Verify your choices in the **Services summary** card on the right
```

Comment:
I think formatting got strange and this should be "4." but got set automatically to "1."